municipal_finance/static/javascript/table.js

```diff
@@ -800,7 +831,22 @@
                 var data = muni_data ? muni_data[functions[f].code] : null;
                 this.renderMuniValues(muni, data, tr);
               }
-            } else {
+            } else if (CUBE_NAME == 'capital_v2') {
+              var data = [];
+              if (muni_data){
+                Object.keys(aggregate_columns).forEach(column => {
+                  var row = "";
+                    for (var f = 0; f < muni_data.length; f++) {
+                      if (muni_data[f]["capital_type.code"] == aggregate_columns[column]["code"]){
+                        row = muni_data[f];
+                      }
+                    }
+                    data.push(row);
+                });
+              }
+              this.renderMuniValues(muni, data, tr);
+            }
+            else {
               this.renderMuniValues(muni, muni_data && muni_data[0], tr);
             }
```

Comment:
The capital cubes have the functions dimension, which currently offers the user the option to disaggregate by government function.

![Screenshot_2022-09-02_13-44-10](https://user-images.githubusercontent.com/235801/188132723-e8a78d86-e3c9-480e-915b-285fc1687c87.png)

I think to get this change out the door, it's ok to disable the function controls for capital_v2 cube. That would _kind of_ make it ok to say `if functions.length OR if cube == capital_v2` because we're then ensuring that we can't have both of those bring true. Perhaps to highlight that things are getting dodgy, please change

```
if (functions.length > 0) {
```

to 

```
if (functions.length > 0 && CUBE_NAME != 'capital_v2') {
```

so this will become 

```
            if (functions.length > 0 && CUBE_NAME != 'capital_v2') {
              for (var f = 0; f < functions.length; f++) {
                var data = muni_data ? muni_data[functions[f].code] : null;
                this.renderMuniValues(muni, data, tr);
              }
            } else if (CUBE_NAME == 'capital_v2') {
```

I see that what we really want is a pivot table rendering component and we don't want to build one ourselves because it's pain.

If you want to be extra cautious, you could add a clause

```
} else if (functions.length > 0 && CUBE_NAME == 'capital_v2') {
    alert("An unexpected error occurred.");
    raise("A user shouldn't be able to disaggregate by function on the capital_v2 cube at the moment");
}
```